packages/frontend-vitals-monitoring/README.md

```diff
@@ -0,0 +1,149 @@
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+     <li><a href="#frontendvitalsinitserver-options">`frontendVitalsInit(server, options)`</a></li>
+  </ol>
+</details>
+
+## About The Project
+
+The GDS Frontend Vitals Signs node package exposes custom metrics (vital-signs) about the health of our Node applications.
+
+The purpose of this package is to make it easy for our FE services to scale in response to demand.
+
+The package is owned by the DI Frontend Capability team, part of the development of this tool involves ongoing discovery with the pods responsible for maintaining the frontend repositories that make up the One Login journey. A stable release of the package is now deployed in production.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+### Installation
+
+1. Install NPM package
+
+   ```sh
+   npm install @govuk-one-login/frontend-vitals-signs
+   ```
+2. Import the package in your Node.js application's startup file (example: app.js or index.js):
+
+   ```js
+   const { frontendVitalsInit } = require("@govuk-one-login/frontend-vital-signs");
+   ```
+
+  [!WARNING] Verify the path to your node module folder is the correct.[!WARNING]
+
+3. In your .env file, add the LOG_LEVEL variable and set it to the desired log level. The   available log levels are:
+
+   - fatal: Logs critical errors that may cause the application to crash.
+   - error: Logs error messages indicating something went wrong.
+   - warn: Logs warnings about potential issues that are not errors but might need attention.
+   - info: Logs informational messages about the application's operation.
+   - debug: Logs debug information useful for development.
+   - trace: Logs detailed information for debugging purposes.
+   - silent: Suppresses all logging output.
+
+  Example:
+
+  Setting the log level to info will log informational messages and errors. To view the logs of our metric object, set it as follows:
+
+  ```dotenv
+  LOG_LEVEL=info
+
+  Example Log:
+
+   ```json
+  {
+    "level": 30,
+    "time": 1722420636051,
+    "pid": 25367,
+    "hostname": "GDS12516",
+    "name": "@govuk-one-login/frontend-vitals-signs",
+    "version": "0.0.1",
+    "requestsPerSecond": {
+      "dynamic": 1,
+      "static": 0.6
+    },
+    "avgStaticResponseTime": 1.5,
+    "avgDynamicResponseTime": 14.5
+  }
+
+   ```
+
+4. Configure the server to use the frontendVitalsInit function. This function initialises the frontend vitals monitoring.
+
+  Example:
+
+   ```js
+
+  const server = app.listen(port, () => {
+    console.log(`Example app listening on port ${port}`);
+  });
+  
+  frontendVitalsInit(server, {
+    interval: 10000, 
+    metrics: ["requestsPerSecond", "avgResponseTime"],
+    staticPaths: ["/assets", "/ga4-assets", "/javascript", "/stylesheets"], 
```

Comment:
Can we include an example of a regex in here as well please? Something like `staticPaths: [/^\/assets\/.*/, "/ga4-assets", "/javascript", "/stylesheets"]`.

Reply:
added